### PR TITLE
Add specimen name to bulk uploader fhir bundle

### LINKED ISSUE #6297: [BUG] bulk uploader FHIR bundle is missing the specimen name
### Background
The bulk uploader FHIR bundle is missing the specimen name

### Action requested
- Add a specimen name to the FHIR if a loinc is provided in the csv
- Add a loinc to the FHIR if a specimen name is provided in the csv

### PR DESCRIPTION
**Pull request by zdeveloper**
# BACKEND PULL REQUEST

## Related Issue

- resolves #6297

## Changes Proposed

- Add a specimen name to the FHIR if a loinc is provided in the csv
- Add a loinc to the FHIR if a specimen name is provided in the csv
- Increase testing

### CODE REVIEW

File: backend/src/test/resources/fhir/bundles-upload-integration-testing.json [hunk 1]
```diff
@@ -1,423 +0,0 @@
-{
```
Comment (zdeveloper) [inline backend/src/test/resources/fhir/bundles-upload-integration-testing.json:1]: This file only contained the first fhir bundle

File: backend/src/test/resources/fhir/bundles-upload-integration-testing.ndjson [hunk 2]
```diff
@@ -0,0 +1,2799 @@
+{
```
Comment (zdeveloper) [inline backend/src/test/resources/fhir/bundles-upload-integration-testing.ndjson:1]: This file contains all 5 generated bundles

**Review by mpbrown — APPROVED** [review]
LGTM!

**Review by nathancrtr — APPROVED** [review]
Looks good, thanks for the speedy fix 🙏

**Pull request merged**